<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff007f;&quot;&gt;Tính xác suất số bóng đèn hư&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Nhập: </string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditD">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditM">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M =</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Chức năng:</string>
    </property>
    <widget class="QPushButton" name="pushButtonTinh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Tính</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLamMoi">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Làm mới</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThoat">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Kết quả:</string>
    </property>
    <widget class="QLabel" name="labelKetQua">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 213, 227);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Trong đó:</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N: số bóng đèn có trong mỗi hộp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>D: Số bóng đèn bị hư có trong mỗi hộp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>M: Số bóng đèn được lấy ngẫu nhiên trong số N bóng đèn</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditN</tabstop>
  <tabstop>lineEditD</tabstop>
  <tabstop>lineEditM</tabstop>
  <tabstop>pushButtonTinh</tabstop>
  <tabstop>pushButtonLamMoi</tabstop>
  <tabstop>pushButtonThoat</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
